### DISCUSSION TITLE
Setting up Hubs Cloud on an existing Domain using a Sub Domain

### DISCUSSION THREAD

**Discussion by Vhive2012** [mozilla/hubs#5480] [Hubs Cloud Questions] [answered]
Good Day,

Has anyone been successful in creating a stack for Hubs Cloud in AWS using a Subdomain on an existing domain that isn't on Route 53? 

I am following [this](https://hubs.mozilla.com/docs/hubs-cloud-aws-existing-domain.html) but can't seem to get it right or working.

I already created a stack with two domains registered in Route 53 so not sure if I need to point the DNS to there or rebuild the stack on the subdomain that isn't registered on Route 53?

Any help would be greatly appreciated.

Thanks.

**Comment by pattersonbl2** [comment, marked as answer]
You would need to rebuild the stack on the subdomain.
